### ISSUE TITLE
Invalid "You are not the owner of this mod" error popup

### ISSUE THREAD

**Issue by Zur13** [Nexus-Mods/web-issues#269]
Bug/error being reported: Invalid "You are not the owner of this mod" error popup
Steps to reproduce:
1. Go to BUGS page of your mod you should have more then 1 page with bug reports
2. Sort bug reports by some column (for example by status column)
3. Try to delete the bug reports using manage menu
4. "You are not the owner of this mod" error dialog window appears

Expected result: Issue removed
Actual result: "You are not the owner of this mod" error dialog appears 

Edit: it seems this error pops up because issue was already removed but still was shown on bugs page after page reload in this case the message should be clarified "Bug report does not exist".

**Comment by mrmason** [comment]
Probably a caching issue - Assigned `PT#152432883`

**Comment by iamtiz** [comment]
I've found and fixed the issue with the wrong error message being displayed, but I've not being able to replicate the problem with the bug report still showing up even after being deleted. Did it happen again in the last few days, by any chance?

**Comment by Zur13** [comment]
That day I've made some cleaning on my mod issues and I closed about 60 issues and after that removed those issues. The closed tag disappeared after page reload and reappeared only after 4-10 minutes the same situation was with removed issues. They are disappeared but was shown again after page reload and completely disappeared only after 4-10 minutes.

I've just tried to do the same and created 30 reports and removed all of them without any problems. They did not reappear after page reload. So it seems that the problem with issues reappear was already solved.
